lib/dialects/oracledb/transaction.js

```diff
@@ -2,9 +2,26 @@ const Transaction = require('../../execution/transaction');
 const { timeout, KnexTimeoutError } = require('../../util/timeout');
 const debugTx = require('debug')('knex:tx');
 
+const supportedIsolationLevels = ['read committed', 'serializable'];
+
 module.exports = class Oracle_Transaction extends Transaction {
   // disable autocommit to allow correct behavior (default is true)
-  begin() {
+  begin(conn) {
+    if (this.isolationLevel) {
+      // There's also a "read only", but that's not really an "isolationLevel"
+      if (!supportedIsolationLevels.includes(this.isolationLevel)) {
+        this.client.logger.warn(
+          'Oracle only supports read committed and serializable transactions, ignoring the isolation level param'
+        );
+      } else {
+        this.client.logger.warn(
+          'Not supporting Oracle transaction isolation right now'
+        );
+        // I tried this, but it didn't work
```

Comment:
Don't have Oracle instance locally, and debugging with Travis is a pain, so yeah, it's fine to skip Oracle for now.